aws/data_source_aws_api_gateway_stage.go

```diff
@@ -0,0 +1,49 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/apigateway"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAwsApiGatewayStage() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsApiGatewayStageRead,
+		Schema: map[string]*schema.Schema{
+			"stage_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"rest_api_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"current_deployment_id": {
```

Comment:
Since the API refers to this as deployment ID, should we remove `current_` from the naming?